src/pages/Tables.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect } from 'react';
```

Comment:
Move this file inside **/src/components/tables** and you can rename it to **UsersTable.jsx**